packages/graphing/configure/src/defaults.js

```diff
@@ -32,6 +32,7 @@ export default {
     includeAxes: true,
     labels: {},
     labelsEnabled: true,
+    dimensionsEnabled: true,
```

Comment:
I think you should remove this from defaults and move the logic inside componentDidMount or constructor. I suggest to check if this value is set to the model and if not, assign the value from the config defaults.